\documentclass[]{beamer}

%\userpackage[latin1]{inputenc}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[french]{babel}
\usepackage{xcolor,colortbl,array,graphicx}
\usepackage{times}
\usepackage{amsthm,amsmath,amsfonts,amssymb,amstext}
\usepackage{lipsum}
\usepackage{hyperref,url}

\title[Etudes informatiques]{On the complexity of SNP Block Partitioning Under the Perfect Phylogeny Model}
% ----------------------
\author{Jens Gramm \inst{1} \and Tzvika Hartman \inst{2} \and Till Nierhoff \inst{3} 
\and  \\  
 roded sharan \inst{4}
 \and  {\color{green} Till Tantau} \inst{5} }
% ----------------------
\institute[UT-BIU-IB-TA-UZL]{
\inst{1}%
Universitat Tubingen, Germany  \and \inst{2}%
Bar-Ilan University, Ramat-Gan, Israel
 \and \inst{3}%
 International Computer Science Institute, Berkeley, USA 
 \and 
 \inst{4}%
 Tel-Aviv University, Israel  \and 
 \inst{5}%
 Universitat zu Lubeck, Germany}
% ----------------------
\date{Workshopon Algorithms in Bioinformatics, 2006}
% ----------------------


% ----------------------
\mode <presentation> {
\usetheme{Darmstadt}
%{Bergen}{PaloAlto}{Berkeley}{Boadilla}{Pittsburgh}{Rochester}
%{Antibes}{Montpellier}{JuanLesPins}{Berkeley}
\usecolortheme{whale}
%{whale} {orchid}{owl}{albatross}{beaver}
\usefonttheme{professionalfonts}
%{professionalfonts} {serif}{structurebold}
%{structureitalicserif}{structuresmallcapsserif}
%\useoutertheme{infolines}
%{infolines} {miniframes}{shadow}{sidebar}
%{smoothbars}{smoothtree}{split}{tree}
\useinnertheme{rounded}
%{circles}{rounded}{rectangles}{inmargin}
\setbeamercovered{transparent=50}
}
\AtBeginSection[]
{
\begin{frame}
<beamer>
\frametitle{\sc Outline}
\tableofcontents[currentsection,currentsubsection]

\end{frame}
}
\begin{document}
	\begin{frame} 
		\titlepage
	\end{frame}
\section{Introduction}
	\subsection{The Model and the Problem}
	\begin{frame}{General formalisation of haplotyping}
	% debut du block
		\begin{block}{\sc inputs}
		\begin{itemize}
			\item A {\color{red} genotype matrix} G.
			\item The {\color{red} rows} of the matrix are 														{\color{red} taxa /individuals.}
			\item The {\color{red} columns} of the matrix are 													{\color{red}SNP sites / characters.}
	
		\end{itemize}
		\end{block}
	% fin du block
	\begin{block}{\sc Outputs}
	\begin{itemize}
	\item A {\color{red} haplotype matrix} G.
	\item Pairs of rows in H {\color{red} explain} the rows of G 
	\item The haplotypes in H are {\color{red} biologically plausible.}
	\end{itemize}
	\end{block}
	
	\end{frame}
	
	\begin{frame}
	\begin{block}{\color {white}}
	\lipsum [1]
	\end{block}
	\end{frame}
	
	\begin{frame}{We can do perfect phylogeny haplotyping efficiently, but \dots}
	\begin{enumerate}
	\item  {\color{red} Data may be missing.}
	\begin{itemize}
	\item <alert@1> {\color {black} This makes the problem NP-complete \dots }
	\item <alert@1> {\color {black} \dots even for very restricted cases}
	\end{itemize}
 {\color{green} Solutions:}
	\begin{itemize}
	\item <alert@1> {\color {black} Additional assumption like the rich data hypothesis.}
	\end{itemize}
	\item {\color{red} No  perfect phylogeny is possible.}
	
	\begin{itemize}
	\item <alert@1> {\color {black} This can be caused by chromosomal crossing-over effects.}
	\item <alert@1> {\color {black} This can be caused by incorrect data.}
	\item <alert@1> {\color {black} This can be caused by multiple mutations at the same sites.}
	\end{itemize}
{\color{green} Solutions:}
	\begin{itemize}
	\item <alert@1> {\color {black} Look for phylogenetic networks.}
	\item <alert@1> {\color {black} Correct for}
	\end{itemize}
	\item {\color{red} Find bloks where a perfect phylogeny is possible}
	\end{enumerate}
	
	\end{frame}
	\begin{frame} {Vibrations longitudinales}{célérité des ondes longitudinales}
	\begin{itemize}
	\item {$[c]$= m/s}
	\item c = célérité du son ou vitesse des ondes longitudinales 
	\item Ordres de grandeur : 
\begin{center}

	$ c_{acier} = \sqrt{\frac{E}{\delta} } = \frac{2.10^{11}}{8.10^{3} }= 5.10^4 = 5000m/s  $ 
	\\
	\hspace{1cm}
\end{center}
	\begin{tabular}{|c|c|c|c|}
	\hline 
	Matériau & c(m/s) & Matériau & c(m/s) \\ 
	\hline 
	PVC mou & 80 & Glace & 3200 \\ 
	\hline 
	Sable sec & 10-300 & Hêtre & 3300 \\ 
	\hline 
	Béton & 3100 & Aliminium & 5035 \\ 
	\hline 
	Plomb & 1200 & Verre & 5300 \\ 
	\hline 
	PVC dur & 1700 & Acier & 5600-5900 \\ 
	\hline 
	Granit & 6200 & Péridotite & 7700 \\ 
	\hline 
	\end{tabular} 
	\\
	
	Rappel : vitesse du son dans l'air :343m/s, dans l'eau : 1480m/s
	\end{itemize}
		\end{frame}
	
	\subsection{The Integrated Approach}
	\begin{frame}{Lipsum 6}
	\lipsum[6]
	\end{frame}
\section{Bad News: Hardness Results}
	\subsection{Hardness of PP-Partitioning of 										Haplotype Matrices}
	\begin{frame}{Lipsum 7}
	\lipsum[7]
	\end{frame}
	\subsection{Hardness of PP-Partitioning of 										Genotype Matrices}
	\begin{frame}{Lipsum 8}
	\lipsum[8]
	\end{frame}
\section{Good News: Tractability Results}
	\subsection{Perfect Path Phylogenies}
	\begin{frame}{Lipsum 9}
	\lipsum[9]
	\end{frame}
	\subsection{Tractability of PPP-Ppartitioning of 			Genotype Matrices}
	\begin{frame}{Lipsum 10}
	\begin{center}
	\lipsum[10]
	\end{center}
	\end{frame}
\end{document}



